spec/system/dashboard_system_spec.rb

```diff
@@ -16,14 +16,14 @@
 
       # rubocop:disable Layout/ExtraSpacing
 
-      # When dashboard loads, ensure that we are on step 1 (Partner Agencies)
+      # When dashboard loads, ensure that we are on step 1 (Storage Locations)
       expect(org_dashboard_page).to     have_getting_started_guide
-      expect(org_dashboard_page).to     have_add_partner_call_to_action
-      expect(org_dashboard_page).not_to have_add_storage_location_call_to_action
+      expect(org_dashboard_page).to     have_add_storage_location_call_to_action
+      expect(org_dashboard_page).not_to have_add_partner_call_to_action
       expect(org_dashboard_page).not_to have_add_donation_site_call_to_action
       expect(org_dashboard_page).not_to have_add_inventory_call_to_action
 
-      # After we create a partner, ensure that we are on step 2 (Storage Locations)
+      # After we create a partner, ensure that we are on step 2 (Partner Agency)
```

Comment:
Please  change this section to reflect the conditions after we do step 1  (i.e. we have created a storage location, and should be seeing the partner call to action, rather than the storage location call to action)

Reply:
Good catch. I have made the changes.

Reply:
Making a very minor tweak to one of the comments. (hardly worth bothering you for)